kasa/iotprotocol.py

```diff
@@ -15,6 +19,8 @@
 class IotProtocol(TPLinkProtocol):
     """Class for the legacy TPLink IOT KASA Protocol."""
 
+    SLEEP_SECONDS_AFTER_TIMEOUT = 1
```

Comment:
minor nit: Maybe `BACKOFF_SECONDS_AFTER_TIMEOUT` since sleep is an implementation detail

Reply:
Done